### PR TITLE
port `ScaleJitter` from detection reference to prototype transforms

### PR DESCRIPTION
**Pull request by pmeier**
```py
import torch
from torchvision.prototype import features, transforms


image = features.EncodedImage.from_path(
    "test/assets/fakedata/logos/rgb_pytorch.png"
).decode()

bounding_boxes = features.BoundingBox(
    [[60, 30, 15, 15]],
    format=features.BoundingBoxFormat.CXCYWH,
    image_size=(100, 100),
    dtype=torch.float,
)

segmentation_masks = torch.zeros((1, 100, 100), dtype=torch.bool)
segmentation_masks[..., 24:36, 55:66] = True
segmentation_masks = features.SegmentationMask(segmentation_masks)

target = dict(
    boxes=bounding_boxes,
    masks=segmentation_masks,
)
sample = image, target

transform = transforms.ScaleJitter(target_size=(100, 100))

torch.manual_seed(0)

for i in range(5):
    transformed_image, transformed_target = transform(sample)
    transformed_bounding_boxes = transformed_target["boxes"]
    transformed_segmentation_masks = transformed_target["masks"]
```

    

I've verified that plugging `image` and `target` in the current reference transform yields the same results.

### CODE REVIEW

**Review by datumbox — APPROVED** [review]
LGTM, thanks!

**Comment by vfdev-5** [comment]
How about adding tests like for other transforms ?

File: test/test_prototype_transforms.py [hunk 1]
```diff
@@ -1102,3 +1102,42 @@ def test_ctor(self, trfms):
...
+
+        transform = transforms.ScaleJitter(target_size=(16, 12), interpolation=interpolation_sentinel)
+        transform._transformed_types = (object,)
+
+        size_sentinel = object()
+        mocker.patch(
+            "torchvision.prototype.transforms._geometry.ScaleJitter._get_params", return_value=dict(size=size_sentinel)
+        )
+
+        inpt_sentinel = object()
```
Comment (vfdev-5) [inline test/test_prototype_transforms.py:1161]: Is it better to use `object()` instead of `MagicMock()` ?

Comment (pmeier) [reply test/test_prototype_transforms.py:1161]: For this purpose they are equivalent. `MagicMock` is useful if the object should have a specific structure or will be transformed by transform. Both things are not the case here.

Comment (vfdev-5) [reply test/test_prototype_transforms.py:1161]: For consistency I'd use `MagicMock`

**Review by vfdev-5 — APPROVED** [review]
LGTM, thanks @pmeier !

**Pull request merged**